### DISCUSSION TITLE
best practice of dealing large file

### DISCUSSION THREAD

**Discussion by wanghaisheng** [smacke/subsync#161] [Q&A] [answered]
ffs video.mp4 -i unsynchronized.srt -o synchronized.srt

such as i got a video duration is 8 hours

**Comment by smacke** [comment, marked as answer]
Hi there, ffsubsync's design is intended to handle this gracefully, since it streams the audio out of the video while reading, meaning that it shouldn't be RAM bound or anything. It would just take a bit longer; maybe up to 2 minutes rather than the typical 20 to 30 seconds. Is it giving you issues on your long video?
